packages/opencensus-core/src/internal/cls-ah.ts

```diff
@@ -1,174 +0,0 @@
-/**
```

Comment:
`cls-hooked` already handles different versions of Node, so I don't think we need this.